avocado/core/multiplexer.py

```diff
@@ -439,7 +439,7 @@ def is_parsed(self):
         """
         return self.variants is not None
 
-    def data_inject(self, key, value, path=None):
+    def data_inject(self, key, value, path=None):   # pylint: disable=E0202
```

Comment:
I have the same version and the same errors. This docstring disables the "hides this method" message, because it's intended. (btw this is a temporary change as in mux PR I'm removing this part)